etc/skydive.yml.default

```diff
@@ -121,6 +121,7 @@ analyzer:
       # list of (sub) probes comprising k8s probe.
       # if list is empty then will resolve to all existing (sub) probes.
       probes:
+        - cluster
```

Comment:
it's enabled by default and will stay like that
why breaking everything .... where the feature ?

Reply:
the change remove the cluster element from being created by default (for both k8s and istio) but also enables no to turn-off the cluster element.

Reply:
cluster is not a k8s resource (its' a child of pod), and in istio we may add workspace (a group of pods defined by selector such as v1) which again is not a k8s resource ... so while most of the nodes we are showing are direct k8s resource this is not true in the general case

Reply:
That still leaves the question : What is the use case for this ? Who would want to remove the creation of a single resource ?

Reply:
why have cluster handled this way:
- you can now disable to enable cluster object; it may be handy when merging k8s to physical to eliminate the cluster node
- all types (cluster included) are handled uniformly as Subprobes (including having cluster code in cluster.go file)
- when we support multiple clusters this will make even more sense ...